calling/src/commonMain/kotlin/com/wire/kalium/calling/CallingConversationType.kt

```diff
@@ -0,0 +1,7 @@
+package com.wire.kalium.calling
+
+enum class CallingConversationType(val value: Int) {
+    OneOnOne(value = 0),
+    Group(value = 1),
+    Conference(value = 2)
+}
```

Comment:
```suggestion
enum class CallingConversationType(internal val avsValue: Int) {
    OneOnOne(avsValue= 0),
    Group(avsValue= 1),
    Conference(avsValue= 2)
}
```

Is `Group` deprecated? Should we use it?

Reply:
I've just asked Chris about it, It's an old style of calling, where there is no server and all clients connect to each other in a mesh.
 we need to check if we can safely remove it